composer.json

```diff
@@ -32,22 +32,22 @@
         "forum": "https://discourse.zendframework.com/c/questions/expressive"
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
         "http-interop/http-middleware": "^0.4.1",
         "roave/security-advisories": "dev-master",
         "zendframework/zend-component-installer": "^1.0",
         "zendframework/zend-config-aggregator": "^1.0",
-        "zendframework/zend-expressive": "^2.0.5",
+        "zendframework/zend-expressive": "^2.1.0",
```

Comment:
I think we should use here `^3.0.0-dev`

Reply:
It didn't work before as there were conflicts with the view renderer packages. I'll try again now that those packages are updated.